src/clients/pizza.ts

```diff
@@ -0,0 +1,95 @@
+import { Effect, Thing } from "../things";
+
+export class PizzaNode {
+  children: Map<string, PizzaNode> = new Map();
+  letters: string;
+  effects: Effect[] = [];
+
+  constructor(letters: string) {
+    this.letters = letters;
+  }
+
+  get options(): Effect[] {
+    return Array.from(this.children.values()).reduce(
+      (acc, curr) => acc.concat(curr.options),
+      this.effects
+    );
+  }
+}
+
+export class PizzaTree {
```

Comment:
i don't love the tree/node construction here but i didn't like it before so it's fine that we didn't totally solve the weird setup of this command. i agree with the decision to decouple it from the wiki stuff it was randomly placed in.